### ISSUE TITLE
Style Guide Deadline

### ISSUE THREAD

**Issue by Ayanda2911** [kevinaangstadt/CS340-S23-Notes#92]
Hey everyone, 

Seeing as it has been some weeks and we have not decided on a the style for our notes, maybe we should set a deadline for that so we can move further along with the project. 

Maybe we could have this done before our next exam on 04/26?

Let me know what you think ?

**Comment by gracecicc** [comment]
I agree, 04/26 sounds like a reasonable deadline to have this finalized.

**Comment by glendalyys** [comment]
I second that!

**Comment by pamaph20** [comment]
Due to the fact that he wants us to have our next PRs by 4/21 this may need to be pushed up. Has anyone stated that they are individually finalizing it or would it be a "group effort".

**Comment by CharlesReinhardt** [comment]
Per #26, I believe @heinzeb7 is in charge of style guide. We can add suggestions to potential features of our style guide on issue #83. Otherwise, I am partial to revising the styles while reviewing our PRs and making revisions.
